test/functional/lua/secure_spec.lua

```diff
@@ -91,14 +93,21 @@ describe('vim.secure', function()
         {2:{MATCH: +}}|
         :lua vim.secure.read('Xfile'){MATCH: +}|
         {3:]] .. msg .. [[}{MATCH: +}|
-        {3:[i]gnore, (v)iew, (d)eny, (a)llow: }^{MATCH: +}|
+        {3:[i]gnore, (v)iew, (d)eny: }^{MATCH: +}|
       ]])
-      feed('a')
+      feed('v')
+      feed(':trust<CR>')
       screen:expect([[
-        ^{MATCH: +}|
-        {1:~{MATCH: +}}|*6
+        ^let g:foobar = 42{MATCH: +}|
+        {1:~{MATCH: +}}|*2
```

Comment:
When the literal contents are known we usually don't use match. It tends to be more readable to write the literal contents, e.g. in this case it would look like (I think):

    {1:~                          }|*2